wagn/lib/wagn/application.rb

```diff
@@ -84,14 +84,14 @@ def paths
         
         paths.add 'files'
         
+        #more consistent to call this deck mod...
         paths.add 'local-mod', :with=>'mod'
         paths['app/models'] = []
         paths['app/mailers'] = []
         
-    
-        # should we have add_deck_paths for these?
-        add_gem_path paths, "lib/tasks",           :with => "lib/wagn/tasks", :glob => "**/*.rake"
+        add_gem_path paths, "lib/tasks",     :with => "lib/wagn/tasks", :glob => "**/*.rake"
         add_gem_path paths, 'gem-assets'
+        add_gem_path paths, 'config/routes', :with => 'lib/wagn/config/routes.rb'
```

Comment:
we discussed this before and moved it here (as in the master branch).  the short of it is that we didn't really like config as a root directory.  it makes sense in an app, but less so in our development framework.

We want to organize the gem directories around teaching Wagn, not around conforming to rails patterns that we may soon break away from.